src/lib.rs

```diff
@@ -1488,14 +1489,23 @@ impl Library {
         let c_filename = CString::new(library_filename)?;
         let mut status = Status::new();
         let inner = unsafe { tf::TF_LoadLibrary(c_filename.as_ptr(), status.inner()) };
+
+        let buf = unsafe {
+            let mut buf = tf::TF_GetOpList(inner);
+            Buffer::<u8>::from_c(&mut buf, false)
```

Comment:
The C API changed the convention for `TF_Buffer` from being heap-allocated to stack-allocated, which makes `Buffer` awkward to use with `TF_GetOpList`.  I think we need something like:

```rust
let buf = unsafe {
    let buf = tf::TF_GetOpList(inner);
    let heap_buf = tf::TF_NewBuffer();
    *heap_buf = buf;
    Buffer::<u8>::from_c(heap_buf, true)
};
```

Reply:
Thanks for this, did not realize the `TF_Buffer` was stack-allocated. I've implemented your suggested method, however I needed to use `owned: false` in `::from_c`, otherwise there are memory address errors.